src/index.ts

```diff
@@ -48,3 +49,22 @@ export const generateBase64String = (length: number): string => {
 export const randInt = (max: number): number => {
   return Math.floor(Math.random() * Math.floor(max));
 };
+
+/**
+ * These two sample functions are wrappers for lodash sample, which can
+ * return undefined. These are guaranteed never to return undefined.
+ * @param fromList
+ */
+export const sampleNum = (fromList: Array<number>): number => {
```

Comment:
This was because TypeScript was complaining about sample when trying to generate data for DSNP messages and ActivityPubs :(

Reply:
P.S.
You can use a generic type and if the Array is typed it can infer correctly:

```suggestion
export const sample = <T>(fromList: Array<T>): T => {
```